cms/templates/index.html

```diff
@@ -552,6 +552,8 @@ <h3 class="title title-3">${_('Need help?')}</h3>
 
   %endif
 </div>
+<%static:renderReact component="HelloWorld"></%static:renderReact>
```

Comment:
So the goal here is that `component` references the named entry point (as specified in the webpack config) of the component you want to render. IMO these named entry points should match the names of the components they're importing.

Reply:
(this is just here for the purposes of making review easier. once folks have given this PR the initial signoff I'll remove it)